maxim/logger/fireworks/helpers.py

```
@@ -0,0 +1,171 @@
+"""Helper functions for Fireworks AI SDK streaming operations.
+
+This module provides helper functions for handling streaming responses from the
+Fireworks AI SDK while maintaining proper integration with Maxim's logging system.
+It handles both synchronous and asynchronous streaming operations, accumulating
+content and usage data to provide complete logging information.
+"""
+
+from typing import Iterator, AsyncGenerator, Generator
```

Ruff (0.11.9)

Import from `collections.abc` instead: `Iterator`, `AsyncGenerator`, `Generator`

Import from `collections.abc`

(UP035)